Makefile

```diff
@@ -42,7 +42,7 @@ proxy:
 
 runner:
 	for target in py312 py311 py310 py39 py38; do \
-		docker build . --no-cache --target $$target --platform=linux/amd64 -f ./docker/Dockerfile.runner -t localhost:5001/beta9-runner:$$target-$(runnerTag); \
```

Comment:
put back

Reply:
I think i added this a while back when i was compiling workers for lambda locally. It dramatically increases compile time when building workers locally but was necessary before adding the github action

Reply:
it's necessary for building the correct arch on m1s / arm hardware

Reply:
this is for the runner image, not for the worker